dace/graph/nodes.py

```diff
@@ -646,6 +666,15 @@ def __init__(self, consume, dynamic_inputs=None):
     def from_json(json_obj, context=None):
         c = Consume("", ['i', 1], None)
         ret = ConsumeEntry(consume=c)
+
+        try:
+            # Set map reference to map exit
+            nid = int(json_obj['scope_exits'][0])
+            exit_node = context['sdfg_state'].node(nid)
+            exit_node.consume = c
+        except IndexError:
+            pass
```

Comment:
Is IndexError always benign?

Reply:
Yes, it's always benign. It's either that the node index of the entry node is smaller than the exit node or otherwise. If it's smaller, it's handled by the node with the larger index.

I'll add a comment.